### ISSUE TITLE
Exception happened when test the example: array.av

### ISSUE THREAD

**Issue by kevenYLi** [killme2008/aviator#272]
**background**:

- Branch: 5.1.2
- OS: mac mojave 10.14.6

**reproduce steps**
- download aviator shell according to [guideline](https://github.com/killme2008/aviator/blob/master/README.md)
- change privileges 
- execute the command: bash avaitor
- clone the codes of aviator
- cd aviator
- execute the command: bash ~/bin/aviator examples/array.av

**stacktrace**

type(a) is : int[]
count(a) is: 4
hello,world,aviator
Exception in thread "main" java.lang.ClassNotFoundException: java.lang.int
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at java.lang.Class.forName0(Native Method)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at java.lang.Class.forName(Class.java:264)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at com.googlecode.aviator.runtime.function.seq.SeqNewArrayFunction.call(SeqNewArrayFunction.java:56)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at Script_1596537804747_0/1639622804.execute0(Unknown Source)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at com.googlecode.aviator.ClassExpression.executeDirectly(ClassExpression.java:65)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at com.googlecode.aviator.BaseExpression.execute(BaseExpression.java:132)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at com.googlecode.aviator.Main.main(Main.java:42)

**array.av**
```
## examples/array.av

let a = seq.array(int, 1, 2, 3, 4);

println("type(a) is : " + type(a));
println("count(a) is: " + count(a));

let s = seq.array(java.lang.String, "hello", "world", "aviator");

println(string.join(s, ","));

let a = seq.array_of(int, 3);
println("type(a) is : " + type(a));
println("count(a) is: " + count(a));

println("before assignment:");
for x in a {
  println(x);
}

for i in range(0, 3) {
  a[i] = i;
}

println("after assignment:");
for x in a {
  println(x);
}


```

**Comment by killme2008** [comment]
Thank you ,it's a bug in `seq.array_of` function ,you can use `seq.array_of(Integer, 3)` to avoid it. I will fix it ASAP.

**Comment by killme2008** [comment]
Fixed in 5.1.3 https://github.com/killme2008/aviator/releases/tag/aviator-5.1.3
